paddle/phi/api/yaml/op_compat.yaml

```diff
@@ -556,6 +553,11 @@
   extra :
     attrs : [bool use_mkldnn = false]
 
+- op : fused_conv2d
```

Comment:
`fused_conv2d` has own operator with extra attributes declared. Is it necessary to add _op_compat_ anyway?

Reply:
Yes, it is necessary.
The extra attributes declared in `fused_conv2d.pbtxt` are used in Pass. The info of extra attributes is getted from `op_compat.yaml` when executor run the kernel, so it is needed to add op_compat for extra attributes.